config/formats.js

```diff
@@ -3431,6 +3431,20 @@ exports.Formats = [
 					this.add('-anim', source, "Tail Slap", target);
 				};
 			}
+			if (move.id === 'hypervoice' && name === 'bumbadadabum') {
+				move.name = 'Open Source Software';
+				move.type = 'Electric';
+				move.basePower = 110;
+				move.accuracy = 95;
+				move.secondaries = [{chance:20, self:{boosts:{spa:-1}}, volatileStatus:'disable'}];
```

Comment:
You can't. The engine only checks `move.secondaries` to figure out secondary chances. If `move.secondary` is defined, `Tools#getMove` sets `move.secondaries = [move.secondary]` (approximately) and at this point in the code, you're modifying a copy of the data retrieved by `getMove`.